### PR TITLE
First base push

### PR DESCRIPTION
**Pull request by mevenG**
# main thing
- added base back-end for expressjs

# db
- added db models
- added db connection

# routes
- added website router

# endpoints
- added template endpoint
- created 1st endpoint to get all users

# passport
- added discord strategy

### CODE REVIEW

File: DiscordStrategy.mjs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import { Strategy as DiscordStrategy } from 'passport-discord';
+import db from './database/dbConfig.mjs';
+
+passport.serializeUser((user, done) => {
+  //console.log("Serialize");
+  done(null, user.id)
+});
+
+passport.deserializeUser(async (id, done) => {
+  //console.log("Deserializing");
```
Comment (Bims-sh) [inline DiscordStrategy.mjs:13]: We can prolly remove the comments :)

Comment (Bims-sh) [reply DiscordStrategy.mjs:13]: ^ Was just testing commenting

**Review by Bims-sh — COMMENTED** [review]
Do we really wanna do raw SQL inside our backend?

**Comment by mevenG** [comment]
> 

I do it this way only to keep 0.5.0 beta working quicker, this **will** be modified once I have the basic thing working correctly

**Comment by Bims-sh** [comment]
If that's the case I'll approve 👍
Let's just fix it later then xD

**Review by Bims-sh — APPROVED** [review]

**Comment by Bims-sh** [comment]
Ooooh the signed commit rule thing

**Comment by Bims-sh** [comment]
@RainOrigami we have to make sure we either all sign commits from now on or we just agree that it's fine without then

**Comment by Bims-sh** [comment]
> @RainOrigami we have to make sure we either all sign commits from now on or we just agree that it's fine without then

If we sign them we'd have to setup GPG keys with our GitHub Accounts

**Pull request closed without merging**